bcpkix/1.68.0.wso2v1/pom.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
+  ~
+  ~ This software is the property of WSO2 Inc. and its suppliers, if any.
+  ~ Dissemination of any information or reproduction of any material contained
+  ~ herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+  ~ You may not alter or remove any copyright or other notice from copies of this content.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.orbit.org.bouncycastle</groupId>
+    <artifactId>bcpkix-jdk15on</artifactId>
+    <packaging>bundle</packaging>
+    <name>bcpkix</name>
+    <version>1.68.0.wso2v1</version>
+    <description>
+        This bundle will represent bouncycastle PKIX 1.68
+    </description>
+    <url>http://wso2.org</url>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+    </distributionManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcpkix-jdk15on</artifactId>
+            <version>1.68</version>
+            <!-- By setting optional false, dependent projects of this orbit bundle will be able to see bcprov-jdk15on library as a transitive dependency.
+             This is required because we are embedding bcprov-jdk15on library inside this orbit bundle without extracting the content,
+             becuase bcprov-jdk15on library is signed.
+             If you embed a dependency and set optional to true, then dependent projects will not be alble to compile their source with this orbit bundle.-->
```

Comment:
Lines seems contain more than 120 char. Shall we split them?

Reply:
Fixed